### PR TITLE
net/bluetooth: Fix the definition in header

### PR DESCRIPTION
**Pull request by Wootak**

### CODE REVIEW

**Comment by seinfra** [comment]
Target : [9c2f6c4551e60cf9f4cd9594cccfa95ff21c469d] - Code Rule Check OK.

**Comment by seinfra** [comment]
Target : [9c2f6c4551e60cf9f4cd9594cccfa95ff21c469d] - Code Rule Check (C++) OK.

**Comment by davidfather** [comment]
@Wootak Please, follow the commit rule. You need to add description.

File: apps/examples/bluetooth/bt_unit_test.c [hunk 1]
```diff
@@ -21,10 +21,15 @@
...
 
-#include <tinyara/bluetooth/bluetooth.h>
+//#include <tinyara/bluetooth/bluetooth.h>
+#include "bt_unit_test.h"
+#include "bluetooth.h"
 
 int bt_unit_test_main(int argc, char **argv)
 {
-	printf("Enter bt_unit_test_main()\n");
+	PRT("Enter\n");
```
Comment (sunghan-chang) [inline apps/examples/bluetooth/bt_unit_test.c]: BT_PRT

Comment (Wootak) [reply apps/examples/bluetooth/bt_unit_test.c]: I removed the logging code in this PR and modify the title and commit message.
Please review.

**Comment by seinfra** [comment]
Target : [dacbd2ac7e07fe12399286b84770c24af557a713] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [dacbd2ac7e07fe12399286b84770c24af557a713] - Code Rule Check OK.

**Pull request merged**